README.md

```diff
@@ -21,9 +21,12 @@ All User Interface interactions should be as simple as possible and easily under
 * docker
 * docker-compose
 
-After installing and configuring those, download [docker-compose.yml](https://raw.githubusercontent.com/projectcaluma/alexandria/master/docker-compose.yml) and run the following command:
+After installing and configuring those, download [docker-compose.yml](https://raw.githubusercontent.com/projectcaluma/alexandria/master/docker-compose.yml) and run the following commands:
 
 ```bash
+# only needs to be run once
```

Comment:
Adapted from the comment in an internal repo, where it sais "# needed for permission handling, only needs to be run once" - maybe you know a better description?

Reply:
That's ok for me. From a dev it can be expected to understand this IMO.